database/descriptors_db.py

```diff
@@ -1,11 +1,11 @@
 from database.db_protocol import DBProtocol
-
+import os
 
 class DescriptorsDB(DBProtocol):
     def __init__(self):
-        table_name = "descriptors"
+        self.table_name = os.getenv("DESCRIPTORS_DB_TABLE", "descriptors")

```

Comment:
```suggestion
        table_name = os.getenv("DESCRIPTORS_DB_TABLE", "descriptors")
```
DBProtocolのなかでself.table_nameをしてるからいらないかも。

Reply:
他のdbも環境変数読めるようにしたほうがいいかな？
環境変数どんなときに使うかイメージ教えてほしいな。

Reply:
test環境、dev環境、prod環境とかでdatabaseを分けるときに使うかな（今回はtableを分けたけど）

Reply:
いいね〜
他のもそうしよう！